data-raw/algorithm.R

```diff
@@ -0,0 +1,5 @@
+create_algorithm_data <- function(path) {
+  algorithm <- readr::read_csv(path, show_col_types = FALSE)
+  usethis::use_data(algorithm, overwrite = TRUE)
+  fs::dir_ls(here::here("data"), regexp = "algorithm")
```

Comment:
Functions as part of a targets pipeline must output the file path if the function code saves to a file.